### DISCUSSION TITLE
Image output folder?

### DISCUSSION THREAD

**Discussion by RayOpaz** [TencentARC/PhotoMaker#141] [General]
Hey just set up PhotoMaker, is there an output folder for renders...cant seem to find one yet? Thanks Ray

**Comment by ETCore7** [comment]
In [Pinokio VM](https://pinokio.computer/) output can always be found in the cache of the app within the Pinokio install folder.
_...\pinokio\api\photomaker2.git\cache\GRADIO_TEMP_DIR_
